front/lib/api/assistant/templates.ts

```diff
@@ -31,15 +31,18 @@ export async function generateMockAgentConfigurationFromTemplate(
     actions: removeNulls([getAction(template.presetAction)]),
     description: template.description ?? "",
     instructions: template.presetInstructions ?? "",
-    generation: {
-      model: {
+    model: {
+      ...({
```

Comment:
same here, ideally we wouldn't need this

Reply:
Updated, good for you? 🙏🏻

Reply:
Much cleaner thank you 🙏